src/components/ProblemAdministration/ProblemAdministration.tsx

```diff
@@ -40,13 +40,19 @@ export const ProblemAdministration: FC = () => {
     if (problem) setSolutions(problem.solution_set)
   }, [problem])
 
+  const {mutate: uploadPoints} = useMutation({
+    mutationFn: (id: string) => {
+      return axios.post(`/api/competition/problem-administration/${id}/upload-points`, {
+        solution_set: solutions,
+      })
+    },
+    onSuccess: () => {
+      refetchProblem()
+    },
```

Comment:
osobne preferujem menej-riadkovy format, ked sa da, ale nepodstatne:
```suggestion
    onSuccess: () => refetchProblem(),
```